internal/librarian/release_init.go

```diff
@@ -117,7 +120,7 @@ func newInitRunner(cfg *config.Config) (*initRunner, error) {
 }
 
 func (r *initRunner) run(ctx context.Context) error {
-	outputDir := r.workRoot
+	outputDir := filepath.Join(r.workRoot, "output")
```

Comment:
I think this line need to restore.

Context: https://github.com/googleapis/librarian/pull/2015/files